src/hashTable.js

```diff
@@ -0,0 +1,60 @@
+export default class HashTable {
+  constructor() {
+    this.storage = []
+    this._length = 0
+    this.max = 10
+  }
+
+  put = (key, value) => {
+    if(typeof(key) === "undefined"){
+      throw("Cannot insert with undefined key!")
+    }
+    const hashIndex = this.hash(key, this.max)
+    this.storage[hashIndex] = value
+    this._length += 1
+  }
+
+  get = (key) => {
+    const hashIndex = this.hash(key, this.max)
+    return this.storage[hashIndex]
+  }
+
+  contains = (key) => {
+    if(this.get(key)){
```

Comment:
Whenever we use the pattern:
```js
if( booleanExpression ) {
  return true
} else {
  return false
}
```
We can instead return the `booleanExpression` directly - in this case:
```js
return this.get( key ) !== undefined
```
Note that I prefer the use of explicit comparison for clarity, and to avoid possible type coercion issues through reliance on "truthy" and "falsey" values in JS.